RELEASE_NOTES.md

```diff
@@ -1,5 +1,78 @@
 # Release Notes
 
+# Staging 2019/06/05
+## Ingest
+### Exporter v0.8.3.rc:be11bc4
```

Comment:
oh I forgot to remove the commit hash beside the version, it shouldn't be included, sorry. Could you remove the :be11bc4 and just retain v0.8.3.rc. Many thanks!

Reply:
I went ahead and pushed those changes. :)